plugin/src/test/resources/ide/language/xml/Example.html

```diff
@@ -5,7 +5,16 @@
     <title><warning descr="MORFOLOGIK_RULE_EN_US">Ttle</warning></title>
 </head>
 <body>
-    <H1>Example header with <warning descr="EN_A_VS_AN">a</warning> error</H1>\
```

Comment:
Huh? What happened to this rule?

Reply:
XML will be fixed lated at another branch